std/haxe/web/Scheme.hx

```diff
@@ -0,0 +1,57 @@
+package haxe.web;
+/**
+* The scheme consists of a sequence of characters beginning with a letter and followed by any combination of letters, digits, plus (+), period (.), or hyphen (-). Although schemes are case-insensitive, the canonical form is lowercase and documents that specify schemes must do so with lowercase letters. It is followed by a colon (:).
+* @class Scheme
+* @module Tamina
+*/
+@:enum abstract Scheme(String) from String to String  {
```

Comment:
IIRC, the reason we have `from String` there is purely pragmatic - we wanted to ease the copy-pasting from JS to Haxe. Conceptually, having `from T` for an enum abstract is wrong, I think. The correct way as you said would be to have a `@:from` method that checks the value, tho I don't really like implicit methods that carry overhead, so I'd just make it a simple static `fromString` method.